Doc 7592 rn

ibsoln opened the pull request:
Commit 1 - prepare infrastructure to handle maintenance releases
Commit 2 - apply 2.8.1 release note changes
Commit 3 - Revised note for 2.8.1 following feedback

ibsoln committed:
DOC-7592 Release Notes 2.8.1
https://issues.couchbase.com/browse/DOC-7592

Commit 1 - prepare infrastructure to handle mintenance releases

rajagp reviewed (changes requested):
Please replace NOTE for 2.8.1 with "If you are currently using 2.8.0 version , we strongly encourage you to upgrade to 2.8.1 as soon as possible. Version 2.8.0 of Android and Java has been replaced with 2.8.1" ...you can tweak it as needed..thats the spirit of message

ibsoln committed:
DOC-7592 Release Notes 2.8.1
https://issues.couchbase.com/browse/DOC-7592

Commit 2 - apply 2.8.1 release note changes

ibsoln committed:
DOC-7592 Release Notes 2.8.1
https://issues.couchbase.com/browse/DOC-7592

Commit 3 - revised note for 2.8.1 release note

ibsoln commented:
> Please replace NOTE for 2.8.1 with "If you are currently using 2.8.0 version , we strongly encourage you to upgrade to 2.8.1 as soon as possible. Version 2.8.0 of Android and Java has been replaced with 2.8.1" ...you can tweak it as needed..thats the spirit of message

@rajagp See commit 3 https://github.com/couchbase/docs-couchbase-lite/pull/346/commits/424646fe08115d6bc4e215febfd5ba1296bc0b90

rajagp commented on modules/ROOT/pages/_partials/commons/common-releasenotes.adoc:
```diff
@@ -22,12 +22,10 @@ ifdef::param-2-8-1[]
-* Couchbase Lite on Android (Java)
-
-If you develop apps using those products, and those apps could potentially access databases created by a mix of Release 2.7 and Release 2.8 versions, then you should apply Release 2.8.1 as soon as practicable.
+*If you have already upgraded to 2.8.0 we strongly recommend that you upgrade to version 2.8.1 at the earliest opportunity.*
```
Prefer suggestion from Amarantha
"Couchbase Lite for Android 2.8.1, released in November 2020, supersedes version 2.8.0 released earlier this year. If you are currently using version 2.8.0, we strongly recommend that you upgrade to version 2.8.1 at the earliest."

amarantha-k committed:
update intro paragraph

amarantha-k committed:
add missing period
